src/audio/module_adapter/module/modules.c

```diff
@@ -137,6 +136,8 @@ static int modules_init(struct processing_module *mod)
 	}
 	md->mpd.out_buff_size = src_cfg->obs;
 
+	int ret;
+
```

Comment:
I know we can technically do this (C99 yay), but style-wise this isn't something we do a lot. Not blocking, but just logging this looks strange to me.

Reply:
@kv2019i I think it's helpful in cases like this one. You define the variable where it's needed. This avoids having a 100 lines of code between the definition and the use where it becomes difficult to verify that the variable is indeed unused all that time (especially if it's some `int i` or even `int ret`) which then prompts the developer to stick in a redundant initialisation because "who knows what that variable is there for."